nncf/api/composite_compression.py

```diff
@@ -144,6 +144,18 @@ def get_state(self) -> List[dict]:
             composite_state.append(child_scheduler.get_state())
         return composite_state
 
+    def calculate_current_compression_level(self) -> List[Dict[str, object]]:
+        """
+        Returns current compression level for the composite compression algorithm.
+        This state contains the state of all children.
+
+        :return" The composite compression current compression level.
+        """
+        result = []
+        for child_scheduler in self._child_schedulers:
+            result.append(child_scheduler.calculate_current_compression_level())
+        return result
+
```

Comment:
Method `calculate_current_compression_loss` for `CompositeCompressionScheduler` never used but can't be avoided as it required for all not composite compression schedulers

Reply:
how do you handle logic with interaction between loss & schedule?

Reply:
I introduced new base compression scheduler method `calculate_current_compression_level` https://github.com/openvinotoolkit/nncf/pull/679/files#diff-254ec4a586e47718d7f361ebc0025b2e2c5f0461d9c27a3729bdf7e485bcaedbR116-R123 which returns current compression level based on current scheduler state. I employ this method in `compression_ctrl.load_state` to get current compression level and then to apply it to compression loss. 

To make compression loss to be loaded from checkpoint one should only change the `compression_ctrl.get_state` method